packages/app/src/runner/event-manager.ts

```diff
@@ -181,27 +181,27 @@ export class EventManager {
       })
     })
 
-    const logCommand = (logId) => {
-      const consoleProps = Cypress.runner.getConsolePropsForLogById(logId)
+    const logCommand = (logIds) => {
```

Comment:
logIds / commandIds are pretty unclear to me - I kept reading this as "an array of log IDs" and an "array of command IDs". I think, since it has just two props, it might be better to split them out as `(testId, logId)` - or if not, at least rethink the variable name.